test/suite/test_hs04.py

```diff
@@ -71,8 +71,14 @@ def get_stat(self, stat):
         stat_cursor.close()
         return val
 
+    def create_key(self, i):
```

Comment:
Why this change is done in this function, I don't see it is used anywhere in it?